.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Build Plugin
+        run: ./gradlew initProject buildFlintPlugin
+        env:
+          GRADLE_OPTS: -Dorg.gradle.daemon=false
+      - name: Build IDE
+        uses: GabrielBB/xvfb-action@v1.2
+        with:
+          run: ./gradlew buildFlintIde
+        env:
+          GRADLE_OPTS: -Dorg.gradle.daemon=false
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body: |
+            TODO Changes in this Release
```

Comment:
If we do proper commit logs we can use that here, but that requires a certain amount of discipline  :')